src/Features/Demo/Demo.hpp

```diff
@@ -16,6 +16,7 @@ class Demo {
     int32_t playbackFrames;
     int32_t signOnLength;
     std::vector<int32_t> messageTicks;
+    int32_t firstPositivePacketTick;
```

Comment:
I'd default this field to -1, just in case you have some really weird demo without any Packet message